test/scapy/layers/ipsec.uts

```diff
@@ -3157,59 +7132,61 @@ except IPSecIntegrityError as err:
     err
 
 #######################################
-= IPv6 / ESP - Transport - AES-CBC - HMAC-SHA1-96
+= IPv6 / ESP - Tunnel - AEAD - AES-GCM-256
+~ crypto_advanced
 
 p = IPv6(src='11::22', dst='22::11')
 p /= TCP(sport=45012, dport=80)
 p /= Raw('testdata')
 p = IPv6(raw(p))
 p
 
-sa = SecurityAssociation(ESP, spi=0x222,
-                         crypt_algo='AES-CBC', crypt_key=b'sixteenbytes key',
-                         auth_algo='HMAC-SHA1-96', auth_key=b'secret key')
```

Comment:
Why change all of these?

Reply:
Again, I have not deleted anything. Just added some tests.